### DISCUSSION TITLE
Build out the "Decline" feature

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Revisions#904] [Feature Requests]
Should we add the the "Decline" option to the regular links?

I'm guessing that "Decline" notifications should wait for PublishPress integration?
<img width="729" alt="revisions-buttons" src="https://user-images.githubusercontent.com/3868028/172236909-bea19c5b-0e77-4205-b677-f55390abf025.png">

**Comment by agapetry** [comment]
https://github.com/publishpress/publishpress-revisions/issues/2009
